code/modules/projectiles/projectile/energy.dm

```diff
@@ -51,8 +51,10 @@
 	SpinAnimation()
 
 /obj/item/projectile/energy/net/on_hit(atom/target, blocked = 0)
-	if(isliving(target) && !locate(/obj/effect/nettingportal) in loc)
-		new/obj/effect/nettingportal(get_turf(target))
+	if(isliving(target))
+		var/turf/Tloc = get_turf(target)
+		if(!locate(/obj/effect/nettingportal) in Tloc)
+			new/obj/effect/nettingportal(Tloc)

```

Comment:
We need to check if there's already a portal on the mob's tile, NOT on the projectile's tile...